src/core/Akka/Pattern/BackoffSupervisor.cs

```diff
@@ -56,7 +56,7 @@ public CurrentChild(IActorRef @ref)
         /// Send this message to the <see cref="BackoffSupervisor"/> and it will reset the back-off. This should be used in conjunction with `withManualReset` in <see cref="BackoffOptionsImpl"/>.
         /// </summary>
         [Serializable]
-        public sealed class Reset
+        public new sealed class Reset
```

Comment:
Poor naming choices here, but hiding the APIs doesn't hurt anything in our tests or the accessibility of the underlying class. This is pure compiler ceremony.